index_originl.html

```diff
@@ -55,6 +55,15 @@ <h3 class="suggestion">Suggestions</h3>
 
     </form>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+    -->
     <script  src="js/index.js"></script>
   </body>
-</html>
```

Comment:
Please, follow standards semantics for html and css. Avoid leaving empty spaces.
You can check the de following article in case you have doubts or need to refresh them.

[https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md](url)

Reply:
- [ ] Hi @HFG43, Please make sure your review includes an explanation, a "why", for the requested change. This will help make the student understand the reason for the changes requested. :pray: